### DISCUSSION TITLE
Log In + Sign Up instead of Sign In option

### DISCUSSION THREAD

**Discussion by edgarasben** [nextauthjs/next-auth#2004] [Ideas]
## Summary 💭

Not sure if this discussion exist somewhere already, let me know and I will detail this more:

Sign In only is a new concept for many users and it would be a bit less confusing if there were two functionalities - Log In and Sign Up. The Sign Up would check the database if user already exist. This is how https://magic.link works too.

Is it possible already and I am not aware of?

**Comment by balazsorban44** [comment]
Could you elaborate a bit more? Please use the template!
